src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -0,0 +1,175 @@
+    ArrayList<HashMap<String, Object>> releases = new ArrayList<HashMap<String, Object>>();
+    for (int i = 78; i > 0; i--) {
+      HashMap<String, Object> release = new HashMap<String, Object>();
+      release.put("releaseName", String.format("M%d", i));
+      release.put("platform", "Android");
+      release.put("releaseManager", releaseManagers.get(i % releaseManagers.size()));
+      release.put("launchCalDeadline", Instant.now());
+      release.put("codeFreezeTime", Instant.now());
+      release.put("launchDate", Instant.now());
+      release.put("buganizerHotlistLink", "b/12345678");
+      releases.add(release);
+    }
+
+    return ResponseHandler.createJsonResponse(Status.OK, releases);
+  }
+
+  /**
+   * Method handling HTTP POST requests.
+   * Exposed at "v1/platforms/{platform}/releases", this endpoint consumes a  
+   * Release object and adds it to the database. Only admins can do this.
+   *
+   * @return Response object containing a status code and a message to be 
+   * displayed to the client.
+   */
+  @POST
+  public Response addRelease(Release release) {
+    // TODO: Implement this method.
+    String message = String.format("Added %s release to the dashboard.", "M79"/*release.releaseName*/);
+    return ResponseHandler.createJsonResponse(Status.OK, message);
+  }
+
+  /**
+   * Method handling HTTP DELETE requests.
+   * Exposed at "v1/platforms/{platform}/releases/{release}", this endpoint
+   * deletes the given Release object from the database. Only admins can do this.
+   *
+   * @return Response object containing a status code and a message to be 
+   * displayed to the client.
+   */
+  @DELETE
+  @Path("{release}")
+  public Response deleteRelease(@PathParam("release") String release) {
+    // TODO: Implement this method.
+    String message = String.format("Deleting %s release from %s platform", release, platform);
+    return ResponseHandler.createJsonResponse(Status.OK, message);
+  }
+
+  /**
+   * Method handling HTTP GET requests.
+   * Exposed at "v1/platforms/{platform}/releases/{release}/sdks", this endpoint
+   * returns a list of the names of all skds enrolled in the given release.
+   *
+   * @return Response object containing a status code and a list of Strings representing
+   * the names of all the sdks enrolled in the given release.
+   */
+  @GET
+  @Path("{release}/sdks")
+  public Response getReleaseSDKs(@PathParam("release") String release) {
+    // TODO: implement this method
+    HashMap <String, ArrayList<String>> releaseSDKs = new HashMap<>();
+    releaseSDKs.put(release, sdks);
+    return ResponseHandler.createJsonResponse(Status.OK, releaseSDKs);
+  }
+
+  /**
+   * Method handling HTTP POST requests.
+   * Exposed at "v1/platforms/{platform}/releases/{release}/sdks", this endpoint 
+   * consumes an SDKRelease object and enrolls it in the given release in the database.
```

Comment:
nit: remove 'in the database'. This applies to all APIs, so its not really required in the comments.